### PR TITLE
Remove lopocs support

### PR DESCRIPTION
**Pull request by gchoqueux**
## Description
* remove lopocs support;
* rename all PoinCloudXxx to PotreeXxx;
* fix Potree debug;
* clean potree examples;

### CODE REVIEW

File: src/Core/PotreeNode.js [hunk 1]
```diff
@@ -80,28 +80,18 @@ class PointCloudNode {
         throw new Error(`Cannot find node with name '${name}'`);
     }
 
-    octreeUrl() {
-        return `${this.baseurl}/r${this.name}.${this.layer.source.extensionOctree}`;
-    }
-
-    nodeUrl() {
```
Comment (zarov) [inline src/Core/PotreeNode.js:87]: Why don't you keep those methods ?

File: src/Layer/PotreeLayer.js [hunk 2]
```diff
@@ -123,31 +123,17 @@ class PointCloudLayer extends GeometryLayer {
...
-                for (const entry of cloud) {
-                    if (entry.table == this.table) {
-                        break;
-                    }
-                    idx++;
-                }
-                this.root.bbox.min.set(cloud[idx].bbox.xmin, cloud[idx].bbox.ymin, cloud[idx].bbox.zmin);
-                this.root.bbox.max.set(cloud[idx].bbox.xmax, cloud[idx].bbox.ymax, cloud[idx].bbox.zmax);
-            }
+            this.supportsProgressiveDisplay = this.source.extention === 'cin';
```
Comment (zarov) [inline src/Layer/PotreeLayer.js]: Could you add parenthesis for readability ?

**Review by zarov — COMMENTED** [review]
Can some things can be kept as `PointCloud*`, for future use ? I don't know the different types of PointCloud and the similarity between them, so I'm not sure about this.

Ping @mbredif

File: src/Core/PotreeNode.js (hunk 1 above)
Comment (gchoqueux) [reply src/Core/PotreeNode.js:87]: Because they were only used once.

File: src/Layer/PotreeLayer.js (hunk 2 above)
Comment (gchoqueux) [reply src/Layer/PotreeLayer.js]: done

**Comment by gchoqueux** [comment]
> Can some things can be kept as `PointCloud*`, for future use ? I don't know the different types of PointCloud and the similarity between them, so I'm not sure about this.
> 
> Ping @mbredif

Point cloud was used to encapsulate `Potree` and `Lopocs`. With this PR all its classes are intended only for the `Potree` format.

**Comment by gchoqueux** [comment]
see @mbredif  [comment](https://github.com/iTowns/itowns/pull/1308#issuecomment-580248741)

**Review by zarov — APPROVED** [review]

File: src/Core/PotreeNode.js (hunk 1 above)
Comment (gchoqueux) [reply src/Core/PotreeNode.js:87]: I add a new commit:
The `PotreeNode` have a `load` function and `url getter`.
This fits better with the future architecture of `itowns`. Where the `command` will perform `node load`.

File: src/Core/PotreeNode.js [hunk 3]
```diff
@@ -80,28 +80,26 @@ class PointCloudNode {
...
+    get url() {
+        return `${this.baseurl}/r${this.name}.${this.layer.source.extension}`;
     }
 
-    loadNode() {
-        return this.layer.source.fetcher(this.nodeUrl(), this.layer.source.networkOptions).then(this.layer.source.parse);
+    load() {
+        // Query octree/HRC if we don't have children potreeNode yet.
+        if (!this.octreeIsLoaded) {
+            this.loadOctree();
```
Comment (zarov) [inline src/Core/PotreeNode.js:94]: How do you handle the `Promise` part ?

Comment (gchoqueux) [reply src/Core/PotreeNode.js:94]: the promise part had never handling in `Provider`. I don't understand why.

**Pull request merged**